### ISSUE TITLE
Feature: Disable pet for player if AFK

### ISSUE THREAD

**Issue by Theepwner** [DSH105/EchoPet#511]
I'm not sure if this plugin is being continued, but if it is: a cool feature would be the ability to set in the config a thing that could toggle off the pet of a player if he/she is AFK for an X amount of time.

Really hope this plugin is continued; no pet plugins can top this.

**Comment by DSH105** [comment]
Yeah, it's being continued, just not *actively* developed at this point in time. I like this idea; I'll see what I can do when i get around to it :).

**Comment by Theepwner** [comment]
Thanks, nice to hear.
